app/Library/General.php

```diff
@@ -1,6 +1,5 @@
 <?php namespace App\Library {
```

Comment:
I didn't see any difference. Not sure why this file is included in this change. Other than the blank line. Please revert it